iphone/package.json

```diff
@@ -19,7 +19,7 @@
 	"minIosVersion": "10.0",
 	"minWatchosVersion": "2.0",
 	"vendorDependencies": {
-		"xcode": ">=9.0 <=12.x",
+		"xcode": ">=11.0 <=12.x",
 		"ios sdk": ">=10.0 <=14.x"
```

Comment:
```suggestion
		"ios sdk": ">=13.0 <=14.x"
```

Reply:
Yeah, I think the min iOS bump can easily be done separately and iOS a bigger issue.